binaries/runtime/src/operator/python.rs

```diff
@@ -47,17 +47,29 @@ pub fn spawn(
             .ok_or_else(|| eyre!("module path has no file stem"))?
             .to_str()
             .ok_or_else(|| eyre!("module file stem is not valid utf8"))?;
-        let module = py
-            .import(module_name)
-            .wrap_err("failed to import Python module")?;
+        let module = py.import(module_name).or_else(|err: pyo3::PyErr| {
+            let trace = err
+                .traceback(py)
+                .expect("PyError should have a traceback")
+                .format()
+                .wrap_err("Traceback could not be formatted")?;
+            Err(eyre!("{trace}"))
+        })?;
         let operator_class = module
             .getattr("Operator")
             .wrap_err("no `Operator` class found in module")?;
 
         let locals = [("Operator", operator_class)].into_py_dict(py);
         let operator = py
             .eval("Operator()", None, Some(locals))
-            .wrap_err("failed to create Operator instance")?;
+            .or_else(|err: pyo3::PyErr| {
+                let trace = err
+                    .traceback(py)
+                    .expect("PyError should have a traceback")
+                    .format()
+                    .wrap_err("Traceback could not be formatted")?;
+                Err(eyre!("{trace}"))
```

Comment:
Looks like this part is duplicated a few times in this file. Perhaps we can factor it out into a separate `convert_py_err` method? To me the ideal solution would be if we could do just `.map_err(convert_py_err)` afterwards.

Reply:
Yep! so I wasn't sure I could recursively get the Python GIL, but you actually can. I called the function `traceback` in accordance to its functionality.